packages/expo-image-picker/ios/MediaHandler.swift

```diff
@@ -251,6 +255,25 @@ internal struct MediaHandler {
 
   // MARK: - utils
 
+  private func replaceFileExtension(fileName: String?, targetExtension: String) -> String? {
```

Comment:
```suggestion
  private func replaceFileExtension(fileName: String, targetExtension: String) -> String? {
```
I think that here we are sure that the `fineName` can't be null.

Reply:
You are right the String? and guard let were unnecessary